sound/soc/intel/boards/sof_sdw.c

```diff
@@ -460,6 +460,7 @@ static void init_dai_link(struct snd_soc_dai_link *dai_links, int be_id,
 			  int (*init)(struct snd_soc_pcm_runtime *rtd),
 			  const struct snd_soc_ops *ops)
 {
+	pr_debug("sof_sdw: create dai link %s, id %d\n", name, be_id);
```

Comment:
can't we use dev_dbg? it's a bit ugly to use pr_debug only here.

Reply:
> can't we use dev_dbg? it's a bit ugly to use pr_debug only here.

I didn't find a way to get `dev` from the existing `init_dai_link` parameters, but yes, I can add a new parameter.